samples/client/petstore/typescript-angular-v10-provided-in-root/builds/with-npm/.openapi-generator/FILES

```diff
@@ -9,12 +9,12 @@ configuration.ts
 encoder.ts
 git_push.sh
 index.ts
-model/./apiResponse.ts
```

Comment:
I think all changed `*/FILES` in your PR should be reverted.
Please also merge the current master again, to fix the build

Reply:
Will do...those were all updated via `generate-samples.sh`, so there may be a bug there, too?

Reply:
It actually looks like `master` has the same changes for all `FILES` now, so hopefully this build will complete...